tardis/io/parsers/csvy.py

```diff
@@ -0,0 +1,21 @@
+import yaml
+import pandas as pd
+
+YAML_DELIMITER = '---'
+def load_csvy(fname):
```

Comment:
please add a documentation stub - using the numpydoc format. Pycharm can help you with that.